### ISSUE TITLE
pdf downloading?

### ISSUE THREAD

**Issue by npit** [venthur/gscholar#24]
Is there a way to also fetch the accompanying pdf files?

**Comment by venthur** [comment]
Nope, but I could imagine an integration with scihub or so. Patches welcome :)
